.ci/openshift-ci/images/entrypoint.sh

```diff
@@ -90,6 +90,31 @@ update_initrd_kernel_modules()
 	( cd "${rootfs_dir}" && find . | cpio -H newc -o | gzip -9 ) > ${initrd_path}
 }
 
+# Manage symlinks on host.
+handle_symlinks() {
+	local create_links_script="/tmp/create_links.sh"
+	cat <<-EOF > "${HOST_MOUNT}/${create_links_script}"
```

Comment:
Would it be simpler to just create this script in-tree and copy it into the correct location before executing it? All the quoting is going to make maintaining it awkward.